service-deployment/clusterconfig-example.yaml

```diff
@@ -44,6 +44,8 @@ clusterinfo:
     docker_username: your_registry_username
     docker_password: your_registry_password
 
+    docker_tag: your_image_tag
```

Comment:
Is it mandatory? What will happen if it's not specified?

Reply:
It's mandatory now. If it's not specified, the program will exist. Should we set thie value as an option?

Reply:
@ydye you can set the tag to `latest` in `docker_build.py` if it's not specified.

Reply:
I think keeping mandatory is a good way. Because cluster administrator should know what they are doing. If administrator want to deploy ```latest``` tag, they can also fill ```latest``` here.

Reply:
@hwuu @abuccts , I set tag as an option. and default value is latest.